tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py

```diff
@@ -18,13 +18,35 @@
 
 import numpy as np
 
-from ..representation import QuestionAnsweringAnnotation
+from ..representation import QuestionAnsweringAnnotation, ExtendedQuestionAnsweringAnnotation
```

Comment:
If this method of associating model outputs with ground truth values is specific to SQuAD, then it is better to name this class `SquadAnnotation`. Otherwise, a more informative name than "Extended..." should be given.

Reply:
the meaning of representation is abstraction from dataset specific format, so we tried to avoid dataset naming in class names (I can not say that format is unique and have relation only to SQUAD). BTW, the previous representation is also used for Squad (it is standart for tf model realization of dataset representation).

In my vision, if the difference in 2 fields, we do not need to create separated class for them at all and can make them to fields of the common representation, so probably we do not need this extra class at all:)

Reply:
@eaidova, yes, we do not need this extra class )